docs/standard/microservices-architecture/secure-net-microservices-web-applications/azure-key-vault-protects-secrets.md

```diff
@@ -56,8 +56,8 @@ In this example, the call to AddAzureKeyVault comes at the end of configuration
 
 ## Additional resources
 
--   **Using Azure Key Vault to protect application secrets***
-    <https://docs.microsoft.com/azure/guidance/guidance-multitenant-identity-keyvault>*
+-   **Using Azure Key Vault to protect application secrets**
+    [*https://docs.microsoft.com/azure/guidance/guidance-multitenant-identity-keyvault*](https://docs.microsoft.com/azure/guidance/guidance-multitenant-identity-keyvault)
```

Comment:
Shouldn't all the links be just normal links, instead of showing the URL? E.g.:

`[**Using Azure Key Vault to protect application secrets**](https://docs.microsoft.com/azure/guidance/guidance-multitenant-identity-keyvault)`